Formula/g/gfxutil.rb

```diff
@@ -0,0 +1,46 @@
+class Gfxutil < Formula
+  desc "Device Properties conversion tool"
+  homepage "https://github.com/acidanthera/gfxutil"
+  url "https://github.com/acidanthera/gfxutil",
+    using:    :git,
+    tag:      "1.84b",
+    revision: "5284a662181ff4b81dd19d1279aedb68725b827f"
```

Comment:
```suggestion
  url "https://github.com/acidanthera/gfxutil.git",
      tag:      "1.84b",
      revision: "5284a662181ff4b81dd19d1279aedb68725b827f"
```
1. You don't need `:using` if the `url` ends in `.git`.
2. Do we need a git checkout? Can we a use a tarball instead?

Reply:
Git checkout was not needed. The formula builds fine with the link to the tarball on the 1.84b tag's page. `url` has been updated on my local `gfxutil.rb`, and a `sha256` added.